engines/saga2/intrface.cpp

```diff
@@ -649,7 +649,7 @@ void CStatusLine::experationCheck() {
 		_waitAlarm.set(_lineQueue[_queueTail].frameTime);
 		_minWaitAlarm.set(_lineQueue[_queueTail].frameTime / 5);
 
-		// copy upto the buffer's size in chars
+		// copy the buffer's size in chars
```

Comment:
The original comment is more correct even if it could be considered a typo.  It's not always the buffer's size, it's up-to the buffer's size (but might be less)